### PR TITLE
Pass the required prop to the combobox FieldLabel

### PR DESCRIPTION
**Pull request by HichamELBSI**
Signed-off-by: HichamELBSI <elabbassih@gmail.com>

### What does it do?

Allow the combobox to display the required state.

### CODE REVIEW

File: packages/strapi-design-system/src/Combobox/Combobox.js [hunk 1]
```diff
@@ -250,7 +251,11 @@ export const Combobox = ({
         {value}
       </VisuallyHidden>
       <Stack spacing={label || hint || error ? 1 : 0}>
-        {label && <FieldLabel id={labelId}>{label}</FieldLabel>}
+        {label && (
+          <FieldLabel required={required} id={labelId}>
```
Comment (gu-stav) [inline packages/strapi-design-system/src/Combobox/Combobox.js:255]: Good catch! Could you also check if the `labelAction` is passed through as `action={labelAction}`? It seems to be missing too.

**Review by udimberto — APPROVED** [review]
### LGTM


### Observation
There's a strange behavior with the Combobox.
It's broking, when does not have any `ComboboxOption` as child.
But I think it's a problem to another time.

### Good job

File: packages/strapi-design-system/src/Combobox/Combobox.js [hunk 2]
```diff
@@ -20,25 +20,27 @@ import { KeyboardKeys } from '../helpers/keyboardKeys';
...
   onCreateOption,
   onInputChange,
   onLoadMore,
-  noOptionsMessage,
-  hasMoreItems,
-  children: nodes,
-  onClear,
+  placeholder,
+  required,
+  value,
```
Comment (HichamELBSI) [inline packages/strapi-design-system/src/Combobox/Combobox.js:43]: ℹ️  Just sorting the props list 😅

File: packages/strapi-design-system/src/Combobox/Combobox.js (hunk 1 above)
Comment (HichamELBSI) [reply packages/strapi-design-system/src/Combobox/Combobox.js]: Nice catch! Pushed 👍🏽

File: packages/strapi-design-system/src/Combobox/Combobox.js (hunk 2 above)
Comment (gu-stav) [reply packages/strapi-design-system/src/Combobox/Combobox.js:43]: 💡  We need an eslint rule for that.

**Review by gu-stav — APPROVED** [review]
Looks good to me. We should keep in mind that I think we need to create a `ComboboxInput` component to be consistent with the rest of the input fields at some point.

Well done - thanks!

**Pull request merged**